webview/src/plots/components/customPlots/CustomPlot.tsx

```diff
@@ -26,7 +26,7 @@ const createCustomPlotSpec = (
     return {}
   }
 
-  if (isCheckpointPlot(plot)) {
+  if (plot.type === CustomPlotType.CHECKPOINT) {
```

Comment:
Switched the function out for an if statement so I could use FILE_SEPARATOR in `custom.ts`, reducing extra arguments being passed around.